### ISSUE TITLE
Stacks: collapse hidden override

### ISSUE THREAD

**Issue by aparajita** [DWilliames/paddy2-beta#6]
Currently, if we mark a stark to collapse hidden layers, it will not hide a layer that is hidden by selecting "None" in a symbol override. For example, I might have a symbol with a stack of icons, and in one instance of the symbol I don't want to show an icon, so I select "None" from the override dropdown. It would be great if stacks would collapse a layer hidden through this means.

**Comment by DWilliames** [comment]
Yeah :) This is in the the backlog for when I do  more work on Symbols.
Right now, I've mainly been working on Stack Groups, alignment and Padding.

Will definitely make sure this will work 👍 Thanks.
